source/administration/devops-command-center.rst

```diff
@@ -1,6 +1,6 @@
-======================
-DevOps Command Center
-======================
+==============================
+Mattermost Incident Management
+==============================
 
 Mattermost Incident Management is the first workflow that we officially support as part of DevOps Command Center. 
```

Comment:
```suggestion
```

Reply:
@itao, I've removed the opening line as rewording it without DevOps Command Center fell flat. LMK your thoughts.